source/features/enable-file-links-in-compare-view.tsx

```diff
@@ -0,0 +1,83 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import GitHubURL from '../github-helpers/github-url';
+
+function handlePRMenuOpening(event: delegate.Event): void {
+	event.delegateTarget.classList.add('rgh-actionable-link'); // Mark this as processed
+
+	// This solution accounts for:
+	// - Branches with slashes in it
+	// - PRs opened from the default branch
+	const dropdown = event.delegateTarget.nextElementSibling!;
+	const viewFile = select<HTMLAnchorElement>('[data-ga-click^="View file"]', dropdown)!;
+	const headBranchUrl = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname;
+	const filepath = dropdown.closest<HTMLDivElement>('[data-path]')!.dataset.path;
+	viewFile.pathname = headBranchUrl + '/' + String(filepath);
+}
+
+function handleCompareMenuOpening(event: delegate.Event): void {
+	event.delegateTarget.classList.add('rgh-actionable-link'); // Mark this as processed
+
+	const dropdown = event.delegateTarget.nextElementSibling!;
+	const viewFile = select<HTMLAnchorElement>('[data-ga-click^="View file"]', dropdown)!;
+	const url = new GitHubURL(viewFile.href);
+	url.assign({
+		branch: location.pathname.replace(/.+:|.+\.{3}/, '')
```

Comment:
Can this be safely replaced with 
```suggestion
		branch: select('[title^="compare"]')!.title.split(' ').pop()
```

It feels like the regex is hard to understand without a comment

Reply:
Is there a reason we cant just take the textContent?

Reply:
Yep, works too! I didn't check.